File: app/src/main/java/ep/vaja3/MainActivity.java
package ep.vaja3;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.view.View;
import android.widget.TextView;

/***
 * TODO: Ohranite stanje števca medtem ko v aplikaciji zarotirate zaslon
 */
public class MainActivity extends AppCompatActivity {

    private TextView textView;
    private int counter = 0;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        textView = (TextView) findViewById(R.id.textView);
    }

    public void increment(View view) {
        counter++;
        textView.setText(String.valueOf(counter));
    }

    /**
     * TODO: Shranite števec v bundle
     *
     * @param outState
     */
    @Override
    protected void onSaveInstanceState(Bundle outState) {
        super.onSaveInstanceState(outState);
    }

    /**
     * TODO: Nastavite stanje števca na vrednost, ki jo preberete iz bundla
     *
     * @param savedInstanceState
     */
    @Override
    protected void onRestoreInstanceState(Bundle savedInstanceState) {
        super.onRestoreInstanceState(savedInstanceState);
    }
}

File: app/src/main/java/ep/vaja3/MainActivity.kt
package ep.vaja3

import android.os.Bundle

import android.view.View
import androidx.appcompat.app.AppCompatActivity
import ep.vaja3.databinding.ActivityMainBinding

/***
 * TODO: Ohranite stanje števca medtem ko v aplikaciji zarotirate zaslon
 */
class MainActivity : AppCompatActivity() {
    private var counter = 0
    private lateinit var binding: ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)
    }

    fun increment(view: View?) {
        counter++
        binding.textView.text = counter.toString()
    }

    /**
     * TODO: Shranite števec v bundle
     *
     * @param outState
     */
    override fun onSaveInstanceState(outState: Bundle) {
        super.onSaveInstanceState(outState)
    }

    /**
     * TODO: Nastavite stanje števca na vrednost, ki jo preberete iz bundla
     *
     * @param savedInstanceState
     */
    override fun onRestoreInstanceState(savedInstanceState: Bundle) {
        super.onRestoreInstanceState(savedInstanceState)
    }
}